### PR TITLE
add portOpenRetries for intermittent zOS timing errors - 295457

### LINKED ISSUE #25053: Reduce intermittent CWWKO0221E errors in some FATs on zOS
We see intermittent CWWKO0221E errors (albeit infrequently) in some FATs when run on zOS and believe this is tied to stress on the VMs in testing.  The error looks like this:
```
[3/31/23 11:17:04:729 GMT] 00000029 com.ibm.ws.tcpchannel.internal.TCPPort                       E CWWKO0221E: TCP Channel defaultHttpEndpoint initialization did not succeed.  The socket bind did not succeed for host * and port 8010.  The port might already be in use.  Exception Message: EDC8115I Address already in use.
```
Tests before and after this one were fine, so it seems like a timing issue.

Mike Ginnick had the suggestion to add `<tcpOptions portOpenRetries="60"/>` to the httpEndpoint to allow for retries and Nathan Rauh suggested we move it into common code.

So changing the common fatTestPorts.xml files from:
```
<httpEndpoint id="defaultHttpEndpoint"
                  host="*"            
                  httpPort="${bvt.prop.HTTP_default}"
                  httpsPort="${bvt.prop.HTTP_default.secure}"/>
```
To:
```
<httpEndpoint id="defaultHttpEndpoint"
              host="*"            
              httpPort="${bvt.prop.HTTP_default}"
              httpsPort="${bvt.prop.HTTP_default.secure}">
  <tcpOptions portOpenRetries="60" />
</httpEndpoint>
```

### PR DESCRIPTION
**Pull request by gjwatts**
Fixes #25053

### CODE REVIEW

**Comment by gjwatts** [comment]
#build

**Review by tabishop — DISMISSED** [review]

File: dev/com.ibm.ws.componenttest/autoFVT-defaults/fatTestPorts.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2017 IBM Corporation and others.
+    Copyright (c) 2023 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.componenttest/autoFVT-defaults/fatTestPorts.xml]: This should be a date range `2017, 2023`

File: dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2017 IBM Corporation and others.
+    Copyright (c) 2023 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml]: This should be a date range `2017, 2023`

File: dev/com.ibm.ws.componenttest/autoFVT-defaults/fatTestPorts.xml (hunk 1 above)
Comment (gjwatts) [reply dev/com.ibm.ws.componenttest/autoFVT-defaults/fatTestPorts.xml]: Fixed, thanks

File: dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml (hunk 2 above)
Comment (gjwatts) [reply dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml]: fixed, thanks

**Review by pnicolucci — APPROVED** [review]

**Comment by gjwatts** [comment]
#libby

**Comment by gjwatts** [comment]
Build ultimately completed successfully except for 2 known defects that are not related to this change.

**Pull request merged**